src/library_html5_webgl.js

```diff
@@ -116,6 +119,9 @@ var LibraryHtml5WebGL = {
           {{{ makeSetValue('attributes', C_STRUCTS.EmscriptenWebGLContextAttributes.renderViaOffscreenBackBuffer, '1', 'i32') }}}
           {{{ makeSetValue('attributes', C_STRUCTS.EmscriptenWebGLContextAttributes.preserveDrawingBuffer, '1', 'i32') }}}
         }
+        if (resultCodePtr) {
+          HEAP32[resultCodePtr >> 2] = {{{ cDefine('EMSCRIPTEN_RESULT_SUCCESS') }}};
+        }
         return _emscripten_sync_run_in_main_thread_2({{{ cDefine('EM_PROXIED_CREATE_CONTEXT') }}}, target, attributes);
```

Comment:
I realize we don't have an error code from the proxied call, but if it returns 0 that does indicate an error, so maybe we should set the result code to *some* error, at least, rather than success, in that case? Or do those errors not matter as much for some reason? (if so a comment would be good)

Reply:
done